scripts/trelloOnboardingBuilder.js

```diff
@@ -83,6 +85,46 @@ var app = new Vue({
                 }
                 return timelineCardPosition;
             }
+            async function reorganizeCards(timelineCardPosition, trelloApiKey, trelloOAuth1) {
+                let reorganizedCardsCount = 0;
+                for (const card of allCardsInLearningPath) {
```

Comment:
@AdriAt360 The algorithm is looks hard, but like again, with an `Array.filter().filter().map().length` it would have looked far more readable.
I'm really saying that as a reviewer PoV, I took too much time for what it is.

Reply:
@pp0rtal I gave a second look at your comments and my code, and I admit I was rude, my apologies 🙏🏽 
I tried to reorder the functions https://github.com/360Learning/360learning.github.io/pull/12/commits/562b26cbaf19c7c3b3db08b3df1b501ad8fc0ef9
can you look at the final code and tell me if it looks better to you?

Reply:
@AdriAt360 I was ok with your code as it was not the platform repo, really. What's important is that tiny tool to be released soon and that you don't waste too much time on comments. 

But love the update, perfect 👌

Reply:
@pp0rtal no worries, the last ticket of the MVP is already done, only waiting for this PR :)